server/db/models/watch.js

```diff
@@ -11,20 +11,27 @@ const Watch = db.define('watch', {
     allowNull: false
   },
   complications: {
-    type: Sequelize.STRING,
+    type: Sequelize.TEXT,
     allowNull: false
   },
   imageUrl: {
     type: Sequelize.STRING,
-    allowNull: false
+    allowNull: false,
```

Comment:
I think you can allowNull here because it will default to the default value if they leave it empty

Reply:
Great catch. It is redundant.